docs/home.md

```diff
@@ -109,6 +109,11 @@ layout: documentation
 
 <div class="row">
   <div class="col-xs-6 col-md-4">
+    <a href="https://www.packtpub.com/big-data-and-business-intelligence/mastering-mesos" class="thumbnail">
```

Comment:
I think the current ordering is alphabetical based on the name of the title, can you place this after "Building Applications..." and before "Mesos in Action..."?